README.md

```diff
@@ -2,25 +2,34 @@
 
 This project provides three elements for the Ayon pipeline:
  * server/ - The Ayon Backend Addon.
- * TODO: client/ - The Ayon (currently OpenPype) desktop integration.
+ * client/ - The Ayon (currently OpenPype) desktop integration.
  * services/ - Standalone dockerized daemons that act based on events (aka `leecher` and `processors`).
 
-## Server
-This is the part that needs to be uplaoded (or copied) to the `/backend/addons/` directory of your Ayon instance, as a helper, you can run the following command:
-`python create_package.py` from the root of this project; which will create a folder `package` and you can copy the **contents** of that into the `addons` folder of your Ayon isntance.
+In order to use this intergrations you'll need to run the `python create_package.py` script which will create a folder with the current version in `ayon-shotgrid/package/shotgrid/0.0.1`, you then have to upload the `shotgrid` directory (included) unto your Ayon instance, in the `/<server root>/addons/shotgrid` path, this should trigger a restart of the server, otherwise do so manually.
```

Comment:
```suggestion
In order to use this intergrations you'll need to run the `python create_package.py` script which will create a folder with the current version in `ayon-shotgrid/package/shotgrid/{addon version}`, you then have to upload the `shotgrid` directory (included) unto your Ayon instance, in the `/<server root>/addons/shotgrid` path, this should trigger a restart of the server, otherwise do so manually.
```

Reply:
intergrations -> integrations
unto -> onto